virtualization/xe-guest-utilities/xe-guest-utilities.yaml

```diff
@@ -0,0 +1,17 @@
+name: xe-guest-utilities
+container:
+  entrypoint: /usr/local/bin/xe-daemon
+  environment:
```

Comment:
this is probably not need as `/sbin` should be already in PATH

Reply:
@frezbo sure! Send me the url and I'll test it for you.

The other binaries are utilities which can't/won't be used, so these aren't required. `xe-linux-distribution` might be interesting to include, but doesn't have support for Talos at the moment. I will look into adding this.

The [udev rule](https://github.com/xenserver/xe-guest-utilities/blob/master/mk/xen-vcpu-hotplug.rules) uses a shell, which won't work on Talos.

Currently the xe-guest-utilities functionality is limited mostly to having support for guest power actions. We're working on implementing more features (like network information) by removing the need of a shell.

Reply:
I'll sent an image link soon

Reply:
`ghcr.io/frezbo/xe-guest-utilities:8.3.0`

Reply:
`xenstore` is also a CLI utility to interact with the store. This isn't required by `xe-daemon`.

I'm not the maintainer of xe-guest-utilities, but see what I can do :) I'll test your image ASAP.